ui/lib/kv/addon/components/kv-delete-modal.hbs

```diff
@@ -0,0 +1,34 @@
+<button type="button" class="toolbar-link" {{on "click" @toggleModal}} data-test-kv-delete={{@mode}}>
```

Comment:
I think the component here should still manage opening/closing the modal since the button is inside it - like I did [here](https://github.com/hashicorp/vault/pull/22313/files)